src/components/hint.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+class Hint extends Component {
+  render() {
+    const hint = this.props.hint;
+    let persons = this.props.person.slice();
+    if (hint.get('status') === "yes please") {
+      let people = "REACT_APP_USE_TEST_DATA" in process.env
+      ? persons
+          .sort(function(a,b) {
+            return Math.random()-Math.random();
```

Comment:
In our discussion, we were having a hard time figuring out if this does what we want. Is there a more conceptually simple way to express this?